docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/ecommerce-tag-template/ecommerce-tag-configuration/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Ecommerce Tag Configuration"
+sidebar_position: 100
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Ecommerce API
+
+Use the native Snowplow Ecommerce API or transitional GA4/UA ecommerce adapter APIs for existing dataLayer implementations using those formats. To get full value from the Snowplow Ecommerce plugin we recommend using the native API when possible.
+
+![](images/01_ecommerce_api.png)
+
+## Tracking Parameters
+
+<Tabs groupId="ecommerceAPI" queryString>
+  <TabItem value="sp" label="Snowplow Ecommerce" default>
+
+![](images/02_sp_tracking_parameters.png)
+
+#### Snowplow Ecommerce Function
+
+In this section you can select the [Snowplow Ecommerce function](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/index.md#functions) to use.
+
+#### Snowplow Ecommerce Argument
+
+In this textbox you can specify the argument to the ecommerce function. This can be a Variable that evaluates to a corresponding object.
+
+#### Additional Tracking Parameters
+
+**Add Custom Context Entities**
+
+Use this table to attach [custom context entities](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/index.md#custom-context) to the Snowplow event. Each row can be set to a Google Tag Manager variable that returns an array of custom contexts to add to the event hit.
+
+**Set Custom Timestamp**
+
+Set this to a UNIX timestamp in case you want to [override the default timestamp](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/index.md#setting-the-true-timestamp) used by Snowplow.
+
+  </TabItem>
+  <TabItem value="ga4" label="GA4 Ecommerce">
+
+![](images/02_ga4_tracking_parameters.png)
+
+#### GA4 Ecommerce Function
+
+In this section you can select the [Google Analytics 4 Ecommerce function](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/index.md#google-analytics-4-ecommerce) to use.
+
+#### GA4 Ecommerce Arguments
+
+**DataLayer ecommerce**
+
+Here you can specify the dataLayer ecommerce variable to use.
```

Comment:
It would be good to clarify if this should be the `ecommerce` object itself, or an object containing the `ecommerce` property. (e.g. Enhanced Ecommerce in the regular template expects the latter but intuitively I would actually provide the former)